medical_center/models/medical_center.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, tools
+from odoo.modules import get_module_resource
+
+
+class MedicalCenter(models.Model):
+    _name = 'medical.center'
+    _description = 'Medical Center'
+    _inherit = 'medical.abstract.entity'
+
+    @api.model
+    def _create_vals(self, vals):
+        vals.update({
+            'is_company': True,
+            'customer': False,
+        })
+        return super(MedicalCenter, self)._create_vals(vals)
+
+    @api.model
+    def _get_default_image(self, vals):
+        res = super(MedicalCenter, self)._get_default_image(vals)
```

Comment:
This is missing tests. Take a look at the patient one - https://github.com/OCA/vertical-medical/blob/10.0/medical/tests/test_medical_patient.py#L120